src/components/common/codeInput/CodeInput.tsx

```diff
@@ -115,13 +115,23 @@ const CodeInput = ({
     >
       Custom Code
     </MenuItem>,
-    valueSets?.map((vs) => {
-      return (
-        <MenuItem key={vs.oid} value={vs.oid} data-testid={`option-${vs.oid}`}>
-          {vs.display_name}
-        </MenuItem>
-      );
-    }),
+    valueSets
+      ?.sort(function (a, b) {
+        var nameA = a.display_name.toUpperCase();
+        var nameB = b.display_name.toUpperCase();
+        return nameA < nameB ? -1 : nameA > nameB ? 1 : 0;
+      })
+      .map((vs) => {
+        return (
+          <MenuItem
+            key={vs.oid}
+            value={vs.oid}
+            data-testid={`option-${vs.oid}`}
+          >
+            {vs.display_name}
+          </MenuItem>
+        );
+      }),
   ];
 
   const codeSystemMenuOptions = () => {
```

Comment:
Why are we not sorting CodeSystem and code Menu options ?

Reply:
Story did mention CodeSystem drop downs, but not codes, may be we can check with POs regarding codes

Reply:
There are dropdowns from Codes, Screenshot from Story. I believe that will be `madie-patient/src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/Codes/Codes.tsx`
![image](https://github.com/MeasureAuthoringTool/madie-patient/assets/57495404/cd5235e3-da3b-4b1c-b977-38264124ab7c)